backend/src/main/java/com/crm/service/impl/CustomerServiceImpl.java

```diff
@@ -33,4 +36,14 @@
 
         return new PageImpl<>(customerResponses, pageable, customerEntities.getTotalElements());
     }
+
+    @Override
+    public void addCustomer(CustomerRequest customerRequest) {
+        try {
+            repository.save(mapper.convertToEntity(customerRequest));
+        } catch (RuntimeException exc) {
```

Comment:
save method throws IllegalArgumentException, anyway we don't need here a try catch. Please check:
![image](https://user-images.githubusercontent.com/33421875/123539765-905d2d80-d73b-11eb-9696-0105182ef6a1.png)
In our case entity will never be null